core/src/main/java/com/yahoo/oak/OakMapBuilder.java

```diff
@@ -120,4 +126,32 @@ public OakMapBuilder(OakComparator<K> comparator,
                 valuesMemoryManager, keysMemoryManager);
     }
 
+
+
+
+
+    public OakHashMap<K, V> buildHashMap() {
```

Comment:
I would name the two building methods: `buildOrderedMap()` and `buildHashMap()`

Reply:
ok